Gemfile

```diff
@@ -3,18 +3,18 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.5'
+# ruby '2.7.5'
```

Comment:
Did you mean to leave this here but commented?

Reply:
@maxkadel thanks actually I meant to remove it since there is a ruby-version file and a .tool-versions file that I just added in this PR

Reply:
@maxkadel I will keep the ruby version in the gemfile.

Reply:
Sounds good - I think either way is fine.